File: jeehyun/개별문제/SW Expert Academy/1204_최빈수 구하기.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

class Solution {
	public static void main(String[] args) throws Exception {
		BufferedReader bf = new BufferedReader(new InputStreamReader(System.in));
		int T = Integer.parseInt(bf.readLine());
		
		for (int test_case = 1; test_case <= T; test_case++) {
			test_case = Integer.parseInt(bf.readLine());
			System.out.println("#" + test_case + " ");
			
			int[] cnt = new int[101];
			String score = bf.readLine();
			StringTokenizer st = new StringTokenizer(score, " ");
			
			while (st.hasMoreTokens()) {
				cnt[Integer.valueOf(st.nextToken())]++;
			}
			
			int answer = 0;
			int maxCnt = 0;
			
			for (int i = 0; i <= 100; i++) {
				if (cnt[i] >= maxCnt) {
					answer = i;
					maxCnt = cnt[i];
				}
			}
			System.out.println(answer);
		}
	}

}

File: jeehyun/개별문제/SW Expert Academy/1204 최빈수 구하기.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int cnt[101];

int main()
{
    int T, k;
    cin >> T;

    for (int test_case = 1; test_case <= T; ++test_case) {
        cin >> test_case;
        cout << "#" << test_case << " ";

        fill_n(cnt, 101, 0);

        for (int i = 0; i < 1000; i++) {
            cin >> k;
            cnt[k]++;
        }

        int answer = 0;
        int maxCnt = 0;

        for (int i = 0; i <= 100; i++) {
            if (cnt[i] >= maxCnt) {
                answer = i;
                maxCnt = cnt[i];
            }
        }
        cout << answer << "\n";
    }
    return 0;
}